src/sentry/replays/post_process.py

```diff
@@ -138,3 +136,47 @@ def _archived_row(replay_id: str, project_id: int) -> dict[str, Any]:
         "started_at": None,
         "is_archived": True,
     }
+
+
+CLICK_FIELD_MAP = {
+    "click_alt": "click.alt",
+    "click_aria_label": "click.label",
+    "click_classes": "click.classes",
+    "click_id": "click.id",
+    "click_role": "click.role",
+    "click_tag": "click.tag",
+    "click_testid": "click.testid",
+    "click_text": "click.text",
+    "click_title": "click.title",
+}
+
+
+def extract_click_fields(item: MutableMapping[str, Any]) -> list[dict[str, Any]]:
+    """
+    pops all of the click fields from the item and returns a list of the individual clicks as objects
+    """
+    click_fields = [
+        "click_alt",
+        "click_aria_label",
+        "click_classes",
+        "click_id",
+        "click_role",
+        "click_tag",
+        "click_testid",
+        "click_text",
+        "click_title",
+    ]
+    click_dict = {}
+    for click_field in click_fields:
+        click_val = item.pop(click_field, [None])
```

Comment:
I think `[]` would be a better default value for this.  The comprehension on line 177 will remove the null item and return an empty `[]`.